### DISCUSSION TITLE
Allow Periods to be set to Every 2 weeks on Specific Day

### DISCUSSION THREAD

**Discussion by Homemediadocker** [firefly-iii/firefly-iii#8902] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

- I would like to have the ability to add a budget or income amount, and set a period to be every 2 weeks. For instance, i get paid every two weeks on Wednesday, and would like to set my transaction period to be for that timeframe.
- I envision this would be a part of the auto budget? Maybe? I am fairly new to the app so I am not sure where the best place would be. 
- I have not seen where firefly can do this itself. Most of the options are for weekly/monthly expenses, and we track every 2 weeks expenses. 

### Solution

This should add the ability to create 2 week periods and budgets

**Comment by JC5** [comment]
Hi, thanks for opening an issue. This will [not be happening](https://docs.firefly-iii.org/explanation/more-information/what-its-not/#changing-the-monthly-cycle-to-another-day-of-the-month). What you see is what you get.

Do yourself a favor and separate your financial planning from your paycheck. Firefly III is opinionated [about this](https://docs.firefly-iii.org/explanation/firefly-iii/background/personal-finances/), and it will not be changed.

**Reply by quincarter** [reply]
Hey thank you for your response! So I do think it is close to what I want. And I had read that page you linked about managing personal finances. We are also on an envelope based system. The pay period frequency is just something I couldn't figure out. Like I get paid every 2 weeks and I just need my paycheck to reflect that in firefly. Could I adapt an automated transaction to reflect a direct deposit every 2 weeks? I am fairly new to the system and just am not sure about the capabilities. 

If not then it's fine. Thanks for your response either way. 

(Also just realized I responded on my other account. Whoops)

**Reply by JC5** [reply]
You could use recurring transactions for that? Should fit your use case!
